frontend/src/components/auth/TeacherSignup/steps/TeacherSignupFive.tsx

```diff
@@ -0,0 +1,18 @@
+import { VStack, Text } from "@chakra-ui/react";
+import React from "react";
+
+const TeacherSignupFive = (): React.ReactElement => {
+  return (
+    <VStack pt={28}>
+      <Text textStyle="header4" textAlign="center" pb={4}>
+        Teacher Sign Up Confirmation
+      </Text>
+      <Text textStyle="subtitle2" textAlign="center">
+        Please check your inbox for a confirmation email so that you are able to
+        login
+      </Text>
```

Comment:
i omitted the `Open Email` button for now bc i'm not sure if there's a direct way to implement this

Reply:
yeah! i think that makes sense. we can probably just omit it altogether, pending confirmation from product.